bindings/core/src/response.rs

```diff
@@ -55,7 +56,7 @@ use crate::{error::Error, OmittedDebug};
 #[non_exhaustive]
 pub enum Response {
     /// Response for:
-    /// - [`GenerateEd25519Addresses`](crate::method::SecretManagerMethod::GenerateEd25519Addresses)
+    /// - [`GenerateEd25519Address`](crate::method::SecretManagerMethod::GenerateEd25519Addresses)
```

Comment:
Mismatch here is weird

Reply:
did you select the right line? this is not a mismatch reponse

Reply:
the mismatch is that the response is multiple addresses for a single address call

Reply:
Fixed